### PR TITLE
"fixture" needs to be "results"

### PR DESCRIPTION
**Pull request by kaapstorm**
As per change to [spec](https://docs.google.com/document/d/1HyHhyAiuec8FR-j3BJEfB60n82MXJ5JGsLavzW6F4oQ/edit#).

@proteusvacuum cc @phillipm @czue

### CODE REVIEW

**Comment by proteusvacuum** [comment]
Do we need to change `case` to `result` too? Looking [here](url), where it says:
```
Fixtures are returned as 
    <results>
        <result .../>
        <result .../>
    </results>```

**Comment by czue** [comment]
oops, sorry, that sounds right.

**Comment by phillipm** [comment]
No, sorry, that was me being sloppy in the spec... I talked with Norman about this to get on the same page.
Everything in the top level tag can be of whatever structure, in this context, casedb entries. We just needed to set the top level tag to be something consistent that made sense when referencing via xpath: `instance('some-query-result')/results/whatever`. `results` always being constant.

**Comment by proteusvacuum** [comment]
:+1:

File: corehq/apps/ota/tests/test_search_claim_endpoints.py [hunk 1]
```diff
@@ -87,7 +87,7 @@ def test_duplicate_claim(self):
     @run_with_all_backends
     def test_search_endpoint(self):
         known_result = (
-            '<fixture id="case">'  # (this is not the case type)
+            '<results id="case">'  # ("case" is not the case type)
```
Comment (phillipm) [inline corehq/apps/ota/tests/test_search_claim_endpoints.py]: Maybe not worth changing, but on the mobile side we don't use `id="case"` for anything

**Pull request merged**